app/models.py

```diff
@@ -554,7 +554,12 @@ def get_inbound_number(self):
 
     def get_default_sms_sender(self):
         default_sms_sender = [x for x in self.service_sms_senders if x.is_default]
-        return default_sms_sender[0].sms_sender
+        if default_sms_sender:
+            return default_sms_sender[0].sms_sender
+        # Right now there is no default_sms_sender if we just go to 'Send messages' and click on a template.
+        # It will blow up.  What should we return if no default sender?  Testing locally, the app seems happy
+        # with anything.
+        return "sns"
```

Comment:
I don't have a good answer for this now, but this is a good question and something we should come back to with a more definitive solution.